src/erik_rullestad_ex/pa02/test_pa02.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Erik Rullestad, Håvard Molversmyr'
+__email__ = 'erikrull@nmbu.no, havardmo@nmbu.no'
+
+
+import src.erik_rullestad_ex.pa02.chutes_simulations as cs
+
+
+class TestBoard:
+    def test_goal_reach_success(self):
+        board = cs.Board()
+        assert board.goal_reached(90)
+
+    def test_goal_reach_fail(self):
+        board = cs.Board()
+        assert not board.goal_reached(70)
+
+    def test_goal_reach_custom_goal(self):
+        board = cs.Board(goal=100)
+        assert not board.goal_reached(90)
+        assert board.goal_reached(100)
+
+    def test_position_adjustment(self):
+        b = cs.Board()
+        assert b.position_adjustment(1) is 39
+
+
+class TestPlayer:
+    def test_move(self):
+        b = cs.Board(ladders={1: 30, 2: 31, 3: 32, 4: 33, 5: 34, 6: 35},
+                     chutes={40: 12, 41: 11, 42: 10, 43: 9})
+        p = cs.Player(b)
+        pos1 = p.position
+        p.move()
+        pos2 = p.position
+        assert pos1 is not pos2
+        assert pos2 - pos1 >= 30
+        assert pos2 >= 1
+        assert not p.get_position() in b.ladders.keys()
+        assert not p.get_position() in b.chutes.keys()
+
+    def test_get_position(self):
+        b = cs.Board()
+        p = cs.Player(b)
+        p.move()
+        assert p.get_position() is p.position
+
+    def test_get_steps(self):
+        b = cs.Board()
+        p = cs.Player(b)
+        p.move()
+        p.move()
+        assert p.get_steps() is 2
```

Comment:
`2` is of type `int`, which is a primitive type, and doesn't need to use `is` for comparison. `==` is more than enough here.